CMakeLists.txt

```diff
@@ -28,7 +33,7 @@ include_directories(
 
 list(APPEND LIBS
     ${IBP_LIBRARY} ${GOP_LIBRARY} ${CZMQ_LIBRARY} ${TOOLBOX_LIBRARY}
-    ${JERASURE_LIBRARY} ${ZMQ_LIBRARY} ${FUSE_LIBRARIES}
+    ${JERASURE_LIBRARY} ${ZMQ_LIBRARY} ${FUSE_LIBRARIES} gssapi_krb5
```

Comment:
That's the benefit of exhaustively testing each platform. I'll know in a bit if it's actually needed or not.